saviSimulation/savi/asl/working.asl

```diff
@@ -154,3 +242,18 @@ targetFar :-
 	<-	thrust(off).
 +!stopMoving.
 		
+/** Dest **/
```

Comment:
The asl templates I'm familiar with typically group all the rules together close to the start of the program. These rules are close to the end of the file.